src/config/config.php

```diff
@@ -316,18 +324,22 @@
                 'unemployment' => env('TAXES_SOUTH_DAKOTA_UNEMPLOYMENT_TAX_RATE', 0.012),
             ],
 
-            'utah' => [
-                'unemployment' => env('TAXES_UTAH_UNEMPLOYMENT_TAX_RATE', 0.03),
+            'texas' => [
+                'unemployment' => env('TAXES_TEXAS_UNEMPLOYMENT_TAX_RATE', 0.0031),
             ],
 
-            'washingtondc' => [
-                'unemployment' => env('TAXES_WASHINGTONDC_UNEMPLOYMENT_TAX_RATE', 0.027),
+            'utah' => [
+                'unemployment' => env('TAXES_UTAH_UNEMPLOYMENT_TAX_RATE', 0.011),
```

Comment:
was utah supposed to change? was 0.03 before

Reply:
Saw there was a 2020 UT folder and assumed I had updated it like Illinois but I had only update the wage base, so this updates the rate also, now